src/pages/dialog/Dialog.module.css

```diff
@@ -0,0 +1,26 @@
+body {
+	background-color: #f1f3f4;
+}
+
+.Dialog {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	margin-top: 10px;
+	margin-bottom: 10px;
+	height: 95vh;
+}
+
+
+/* overflow property doesnt works perfectly... */
+.DialogField {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
```

Comment:
in Firefox if remove this line - scroll work perfect, otherwise some messages in top of list is hidden

Reply:
ty so much!!! google chrome works perfectly either -1 problem)))